### PR TITLE
changed to cyan color

### PR DESCRIPTION
**Pull request by NitroPlum**
I made the colors better

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,35 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-  </head>
-  <body style="background: red">
```
Comment (NitroPlum) [inline index.html:9]: thisis bad

File: index.html [hunk 2]
```diff
@@ -5,35 +5,8 @@
...
-    <h1
-      style="
-        font-family: 'Comic Sans MS', 'Comic Sans';
-        color: rgb(82, 255, 47);
-      "
-    >
-      Hell World Hell World Hell World Hell World Hell World Hell World Hell
-      World Hell World Hell World Hell World Hell World Hell World Hell World
-      Hell World Hell World Hell World Hell World Hell World Hell World Hell
```
Comment (NitroPlum) [inline index.html:18]: looks great

File: index.html [hunk 3]
```diff
@@ -5,35 +5,8 @@
...
-      World Hell World Hell World Hell World Hell World Hell World Hell World
-      Hell World Hell World Hell World Hell World Hell World Hell World Hell
-      World Hell World Hell World Hell World Hell World Hell World Hell World
-      Hell World Hell World Hell World Hell World Hell World Hell World Hell
-      World Hell World Hell World Hell World Hell World Hell World Hell World
-      Hell World Hell World Hell World Hell World Hell World Hell World Hell
-      World Hell World Hell World Hell World Hell World Hell World Hell World
```
Comment (Cozimo64) [inline index.html:25]: needs heaven

**Review by Cozimo64 — COMMENTED** [review]
is bad

**Pull request merged**